karspexet/settings.py

```diff
@@ -129,3 +131,18 @@
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
 STATIC_URL = '/static/'
+
+try:
+    with open(BASE_DIR + "/env.json") as env_json:
+        ENV = json.load(env_json)
+except FileNotFoundError:
+    import sys
+    sys.stderr.write("""
+    ================================================================================
+    No env.json settings file found.
+
+    Try starting with the sample one:
+
+    cp env.json.sample env.json
+    """)
+    sys.exit(1)
```

Comment:
You should use the builtin `exit` function since I'm not sure if `sys.exit` calls any exit handlers. (Or it's just an alias, but it does look nicer with just `exit`.)